### PR TITLE
creating comment model

### PR DESCRIPTION
**Pull request by JaneGH**
added comments model, routs get and create

### CODE REVIEW

File: src/interfaces/commentInterfaces.ts
Comment (chukwuemeka-fincra) [inline src/interfaces/commentInterfaces.ts:1]: typically you'd have this in Comment.ts model, you can add the code there no need for additional file

File: src/validations/commentValidation.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import Joi from "joi";
+import { bookIdSchema } from "./bookValidation";
+
+export const commentJoiSchema = Joi.object({
+  user: Joi.string(),
```
Comment (chukwuemeka-fincra) [inline src/validations/commentValidation.ts]: is user an input?
this should be gotten from the auth middleware

File: src/interfaces/commentInterfaces.ts
Comment (JaneGH) [reply src/interfaces/commentInterfaces.ts:1]: fixed

File: src/validations/commentValidation.ts (hunk 1 above)
Comment (JaneGH) [reply src/validations/commentValidation.ts]: it is not user input. Deleted this line

**Pull request merged**